doc/changes/changes_1.6.0.md

```diff
@@ -0,0 +1,37 @@
+# 1.6.0 - 2025-06-26
+
+## Summary
+
+### Links in the Documentation 
+This version of the PTB adds nox tasks to check links present in our documentation:
+
+    docs:link - List all the links within the documentation
```

Comment:
I might be nitpicking here, but the target does not scream "list the links" in my eyes. If I saw this without context I would expect something like "somehow link documents together".

But is is probably a bit late for my comment at this point, so feel free to ignore it.

Reply:
this wouldn't be hard to change. what would you think about `link:list` and `link:check` for the nox task names?